src/remediations/fifi.integration.js

```diff
@@ -153,10 +153,10 @@ describe('FiFi', function () {
             // use impl version
             base.getSandbox().stub(dispatcher, 'getConnectionStatus').callsFake(dispatcher_impl.getConnectionStatus);
 
-            // replace doHttp() with our own function...
-            const spy = base.getSandbox().stub(Connector.prototype, 'doHttp');
+            // replace request() with our own function...
+            const http_request = require('../util/request');
```

Comment:
Shouldn't requirements be at the top of the module? Or is this intentionally here?